src/test/java/th/ac/kmitl/soa/group2/models/supplychaintradetransaction/EmailUriModelTest.java

```diff
@@ -11,9 +11,9 @@
     private static final EmailUriModel model = new EmailUriModel(email);
 
     private final String xml =
-        "<EmailUriModel>"+
+        "<ram:EmailURIUniversalCommunication>"+
             "<ram:URIID>"+ email + "</ram:URIID>" +
```

Comment:
xml เหลือแค่บรรทัดนี้พอครับ ไม่ต้องมี root element `<ram:EmailURIUniversalCommunication>`
แล้วก็ลบ `@JacksonXmlRootElement(localName = TELEPHONE_UNIVERSAL_COMMUNICATION)` ออกจากตัว model
เพราะเราจะไม่ใช้ model นี้เป็น root ของ xml ที่จะ generate ให้ client อยู่แล้ว

Reply:
แล้วก็ใช้ tag function ตาม #56

Reply:
เอาบรรทัด 14 ออกใช่มั้ยครับ แล้วแบบนี้ตอนเทสนี่ควรทำยังไงอะครับ

Reply:
ตามนี้ครับ

https://github.com/it-kmitl-2018/group2-backend/blob/69422dd6efb75106b562b30932e5e288cac27f00/src/test/java/th/ac/kmitl/soa/group2/models/HeaderNoteModelTest.java#L19

Reply:
Nice.